map/README.md

```diff
@@ -14,6 +14,16 @@ You may find instructions on how to contribute data to the map [here](src/data/R
 
 In the project directory, you can run:
 
+### Build @reach4help/model dependency:
+
+`cd ../model` `yarn build`
+
+Without this step, you will see this error:
+
+```
+Module not found: Can’t resolve ‘@reach4help/model/lib/markers’ in ‘/Users/ethanstrominger/projects/reach4help/map/src/data’
+```
```

Comment:
Where is this pathname coming from?  It looks like it is a problem in @ethanstrominger 's local environment.  
```py
python delhi_parse.py``` 
works fine on my machine

Reply:
Oops, I did not update that to ~. I had this error on my local machine as well.